src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -1,56 +1,61 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <!--
   For more information on how to configure your ASP.NET application, please visit
   http://go.microsoft.com/fwlink/?LinkId=169433
   -->
 <configuration>
   <system.web>
-    <compilation debug="true" targetFramework="4.5.2" />
-    <httpRuntime targetFramework="4.5.2" />
+    <compilation debug="true" targetFramework="4.5.2"/>
+    <httpRuntime targetFramework="4.5.2"/>
     <httpModules>
-      <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" />
+      <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web"/>
     </httpModules>
-    <customErrors mode="Off" />
-    <sessionState mode="Off" />
+    <customErrors mode="Off"/>
+    <sessionState mode="Off"/>
   </system.web>
   <appSettings>
     <!--
     <add key="Local.Lucene.Directory" value="C:\nuget\SearchData\Lucene-v2v3" />
     <add key="Local.Data.Directory" value="C:\nuget\SearchData\ng-search-data" />
     -->
 
-    <add key="Storage.Primary" value="" />
-    <add key="Search.IndexContainer" value="" />
-    <add key="Search.DataContainer" value="" />
-    <add key="Search.IndexRefresh" value="300" />
-    <add key="Search.RegistrationBaseAddress" value="http://api.nuget.org/v3/registration0/" />
-    <add key="owin:AppStartup" value="NuGet.Services.BasicSearch.Startup" />
+    <add key="Storage.Primary" value=""/>
+    <add key="Search.IndexContainer" value=""/>
+    <add key="Search.DataContainer" value=""/>
+    <add key="Search.IndexRefresh" value=""/>
+    <add key="Search.RegistrationBaseAddress" value="http://api.nuget.org/v3/registration0/"/>
+    <add key="Search.AuxiliaryDataRefreshRateSec" value="3600"/>
```

Comment:
We could move these values to `DefaultValue` tags on the `IndexingConfiguration` and `BasicSearchConfiguration` objects, e.g. [like this](https://github.com/NuGet/NuGet.Services.Metadata/pull/111/files#diff-4aac902fa9451de88d2bd72d7afa3d0fR24).